lib/filedrop.ts

```diff
@@ -106,13 +128,27 @@ export class FileDropElement extends HTMLElement {
     this.setAttribute('accept', val);
   }
 
+  get multiple() : string | null {
+    return this.getAttribute('multiple');
```

Comment:
Shouldn’t this be `this.hasAttribute('multiple')`?

Reply:
I just need it to return what ever is on the attribute - liked disabled='fart', still has disabled set.